--- a/boot.cmd	2018-10-30 12:19:09.903195099 +0300
+++ b/boot.cmd	2018-10-30 21:15:49.258024388 +0300
@@ -37,7 +38,7 @@
     env import -t ${load_addr} ${filesize}
 fi
 
-setenv bootargs "console=${console} earlyprintk root=${rootdev} rw rootwait resize.root=yes panic=10 loglevel=${loglevel}"
+setenv bootargs "console=${console} earlyprintk root=${rootdev} rw rootwait fsck.repair=yes panic=10 loglevel=${loglevel}"
 
 # Load DT file
 load mmc ${device} ${fdt_addr_r} ${prefix}${fdtfile}
